src/core/CSSLint.js

```diff
@@ -184,6 +185,29 @@ var CSSLint = (function(){
         // normalize line endings
         lines = text.replace(/\n\r?/g, "$split$").split("$split$");
 
+        var ignoreStart = null,
+            ignoreStop = null;
+        CSSLint.Util.forEach(lines, function (line, lineno) {
+            // Keep oldest, "unclosest" ignore:start
+            if(null === ignoreStart && line.match(/\/\*[ \t]*csslint[ \t]+ignore:start[ \t]*\*\//i)) {
+                ignoreStart = lineno;
+            }
+
+            if(line.match(/\/\*[ \t]*csslint[ \t]+ignore:stop[ \t]*\*\//i)) {
```

Comment:
Just a nit, but I usually think of these as start/end, not start/stop.  (So, more declarative, less imperative…  ;)